### PR TITLE
[bluecoat] Upgrade ECS to 8.0.0

### PR DESCRIPTION
**Pull request by r00tu53r**
Automated PR.

Upgrades ECS to 8.0.0.

### CODE REVIEW

File: packages/bluecoat/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.0.0"
```
Comment (adriansr) [inline packages/bluecoat/changelog.yml:2]: versions >= 1.0.0 should be reserved for when the package is released as _GA._ Currently it's still _experimental._

File: packages/bluecoat/manifest.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-version: 0.6.1
+version: 1.0.0
 description: Collect director logs from Blue Coat devices with Elastic Agent.
 categories: ["network", "security"]
 release: experimental
 license: basic
 type: integration
 conditions:
-  kibana.version: "^7.14.1 || ^8.0.0"
+  kibana.version: "^7.16.0 || ^8.0.0"
```
Comment (adriansr) [inline packages/bluecoat/manifest.yml:11]: Same as with other packages. Do we really need to do this?

**Review by adriansr — CHANGES_REQUESTED** [review]

File: packages/bluecoat/data_stream/director/fields/ecs.yml [hunk 3]
```diff
@@ -153,7 +151,7 @@
 - external: ecs
   name: process.pid
 - external: ecs
-  name: process.ppid
+  name: process.parent.pid
```
Comment (efd6) [inline packages/bluecoat/data_stream/director/fields/ecs.yml:154]: Breaking change?

File: packages/bluecoat/manifest.yml [hunk 4]
```diff
@@ -1,14 +1,14 @@
 format_version: 1.0.0
 name: bluecoat
 title: Blue Coat Director Logs
-version: 0.6.1
+version: 0.6.2
```
Comment (efd6) [inline packages/bluecoat/manifest.yml:4]: 0.7.0?

Comment (r00tu53r) [reply packages/bluecoat/manifest.yml]: fixed. Thanks!!

File: packages/bluecoat/data_stream/director/fields/ecs.yml (hunk 3 above)
Comment (r00tu53r) [reply packages/bluecoat/data_stream/director/fields/ecs.yml:154]: That's right!

**Review by marc-gr — APPROVED** [review]

**Comment by adriansr** [comment]
Replaced with #2576

**Pull request closed without merging**